### PR TITLE
fix(deploy): 'portal deploy <workspace>' overwrites upstream content

### PR DESCRIPTION
**Pull request by HenriPro**
Adds wipe to deploy

### CODE REVIEW

File: bin/commands/deploy.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ const Workspace_1 = require("../core/Workspace");
 const RestClient_1 = require("../core/HTTP/RestClient");
 const FileRepository_1 = require("../core/HTTP/Repositories/FileRepository");
 const FileResource_1 = require("../core/HTTP/Resources/FileResource");
+const wipe_1 = require("./wipe");
```
Comment (nateslo) [inline bin/commands/deploy.js]: any reason for this being named `wipe_1` as opposed to `wipe`?

Comment (Jlawlzz) [reply bin/commands/deploy.js]: Its the compiled JS

Comment (nateslo) [reply bin/commands/deploy.js]: derp

Comment (nateslo) [reply bin/commands/deploy.js]: Ah thx, didn't realize I was looking at compiled

**Review by nateslo — APPROVED** [review]

**Pull request merged**